cmd/telegraf/telegraf.go

```diff
@@ -54,6 +54,7 @@ var fUsage = flag.String("usage", "",
 	"print usage for a plugin, ie, 'telegraf -usage mysql'")
 var fService = flag.String("service", "",
 	"operate on the service")
+var fRunAsConsole = flag.Bool("run-as-console", false, "run as console application (windows only)")
 
 // Telegraf version, populated linker.
 //   ie, -ldflags "-X main.version=`git describe --always --tags`"
```

Comment:
Perhaps the usage string should be updated to contain the new `console` flag?

Reply:
Yeah just rename the flag and we can merge.